src/resources/Organizations/Organization.ts

```diff
@@ -73,6 +73,19 @@ export default class Organization extends Resource {
         return this.api.post(`${Organization.baseUrl}/${organizationId}/resume`);
     }
 
+    updateAdditionalInformation(
+        additionalInformationObj: Record<string, Record<string, string | boolean | number> | string | number | boolean>
```

Comment:
:feelsgood: what's that thing? 😂

Reply:
Do you mean the record type?

I didn't want to add the `any` type 😆 

it was to make sure we can add another object that the values could either be a string, boolean, number similar to the test 🙈 

```
{
      "trialProgress": {
          completedSource: true,
      },
       "someOtherObject": {
          helloWorld: "magic",
      },
 }
```

Reply:
hmm good point, I guess to be the most flexible in this case, would `unknown` be a better option?

I feel like 1st and 2nd level would be okay for now

Reply:
Interesting, when I try to type it as `Object` I get a linter error saying to use `object` instead.

But when I try `object` it gives en a linter error saying `Don't use object as type. The object type is currently hard to use` 🤔 

Would `Record<string, any>` be better? The call itself is a simple `{}` plain object so I wasn't sure exactly how to handle that

Reply:
Just answering here for the record (badum tss), but yeah linter knows best, so let's go with its suggestions. i.e. `Record<string, unknown>`